.github/workflows/jest.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           - 5432:5432
     env:
       MM_DEV: 1
+      MM_ENV: test
```

Comment:
Looks like you went back to using the hardcoded moduleManifest. Still need this?